compose/ui/ui/src/uikitInstrumentedTest/kotlin/androidx/compose/ui/test/UIKitInstrumentedTest.kt

```diff
@@ -123,15 +140,39 @@ internal fun runUIKitInstrumentedTest(
  * the application setup.
  */
 @OptIn(ExperimentalForeignApi::class)
-internal class UIKitInstrumentedTest {
+internal class UIKitInstrumentedTest(
+    private val useHostingView: Boolean
+) {
+    companion object {
+        fun delay(timeoutMillis: Long) {
+            val runLoop = NSRunLoop.currentRunLoop()
+            runLoop.runUntilDate(NSDate.dateWithTimeIntervalSinceNow(timeoutMillis.toDouble() / 1000.0))
+        }
+
+        fun waitUntil(
+            conditionDescription: String? = null,
+            timeoutMillis: Long = 5_000,
```

Comment:
Let's convert it to `Duration` to have `XXX.milliseconds` outside for self-documentation